From 1d8b816df0375f64c6c1ea6169a33e631f380082 Mon Sep 17 00:00:00 2001
From: Marek 'marx' Grac <mgrac@redhat.com>
Date: Thu, 19 Feb 2015 16:05:11 +0100
Subject: [PATCH] fencing: Add --ipport for fence_ilo and fence_ipmilan

These two were not covered by default settings which expects that agent
have one of web/telnet/ssh/snmp in device_opt.
---
 fence/agents/ilo/fence_ilo.py         | 2 +-
 fence/agents/ipmilan/fence_ipmilan.py | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/fence/agents/ilo/fence_ilo.py b/fence/agents/ilo/fence_ilo.py
index 46650bd..130f27e 100644
--- a/fence/agents/ilo/fence_ilo.py
+++ b/fence/agents/ilo/fence_ilo.py
@@ -65,7 +65,7 @@ def define_new_opts():
 		"order" : 1}
 
 def main():
-	device_opt = ["ipaddr", "login", "passwd", "ssl", "notls", "ribcl"]
+	device_opt = ["ipaddr", "login", "passwd", "ssl", "notls", "ribcl", "web"]
 
 	atexit.register(atexit_handler)
 
diff --git a/fence/agents/ipmilan/fence_ipmilan.py b/fence/agents/ipmilan/fence_ipmilan.py
index 0d926af..940b4be 100644
--- a/fence/agents/ipmilan/fence_ipmilan.py
+++ b/fence/agents/ipmilan/fence_ipmilan.py
@@ -123,7 +123,7 @@ def define_new_opts():
 def main():
 	atexit.register(atexit_handler)
 
-	device_opt = ["ipaddr", "login", "no_login", "no_password", "passwd",
+	device_opt = ["ipaddr", "ipport", "login", "no_login", "no_password", "passwd",
 		"lanplus", "auth", "cipher", "privlvl", "sudo", "ipmitool_path", "method"]
 	define_new_opts()
 
-- 
1.9.3

